lib/plugins/console/index.js

```diff
@@ -67,6 +67,14 @@ module.exports = function(ctx) {
     ]
   }, require('./publish'));
 
+  console.register('unpublish', 'Moves a published post from _posts to _drafts folder.', {
+    usage: '[layout] <filename>',
+    arguments: [
+      {name: 'layout', desc: 'Post layout. Use post, page, draft or whatever you want.'},
+      {name: 'filename', desc: 'Post filename. "hello-world" for example.'}
+    ]
+  }, require('./publish'));
```

Comment:
It seems wrong. Isn't it?

```suggestion
  }, require('./unpublish'));
```